content/applications/websites/ecommerce/products/cross_upselling.rst

```diff
@@ -0,0 +1,80 @@
+===========================
+Cross-selling and upselling
+===========================
+
+Any sales process is an opportunity to maximize revenues. **Cross-selling and upselling** are sales
+techniques consisting in selling a customer something in addition to the product or service they
+were originally shopping for. It is a great way to maximize the value of each one of your customers.
+
+Cross-selling and upselling can be done via three different features:
+
+- :ref:`Optional products <cross_selling/optional>` upon **adding to cart**;
+- :ref:`Accessory products <cross_selling/accessory>` on the **checkout page**;
+- :ref:`Alternative products <cross_selling/alternative>` on the **product page**.
+
+Cross-selling
+=============
+
+.. _cross_selling/optional:
+
+Optional products
+-----------------
+
+**Optional products** are suggested when customers click :guilabel:`Add to cart`, either from the
+**product page** or **catalog page**. Upon clicking, a pop-up window opens with the
+:guilabel:`Optional Products` displayed in the :guilabel:`Available Options` section.
+
+To enable :guilabel:`Optional Products`, go to :menuselection:`Website --> eCommerce --> Products`,
+select a product, go to the :guilabel:`Sales` tab, and enter the products you wish to feature in the
+:guilabel:`Optional Products`.
+
+.. tip::
+   You can also access the :guilabel:`Sales` tab by selecting a product on your **main shop page**,
```

Comment:
technically, they can access the **product form** (which contains the sales tab, but still) -> that's nitpicking